AmplifyPlugins/API/AWSAPICategoryPlugin/Support/Utils/AppSyncJSONValue+toJSONValue.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+import AppSyncRealTimeClient
+
+extension AppSyncJSONValue {
+    static func toJSONValue(_ json: AppSyncJSONValue) -> JSONValue {
```

Comment:
this looks good to me, i'm not sure if there's any other way around this? maybe there's a way to just decode the `payload?["errors"]` directly to `[JSONValue.self]` instead of this